dbmigrate-lib/src/drivers/sqlcipher.rs

```diff
@@ -0,0 +1,124 @@
+use url::Url;
+
+use sqlcipher_client::Connection;
+
+use super::Driver;
+use errors::{Result, ResultExt};
+
+const SQLCIPHER_DEFAULT_CIPHER: &'static str = "AES-256-CBC";
+
+#[derive(Debug)]
+pub struct Sqlcipher {
+    conn: Connection,
+}
+
+impl Sqlcipher {
+    pub fn new(url: &str) -> Result<Sqlcipher> {
+        // Expect a URL in the form:
+        // sqlcipher://user:password@host/path/to/file.db
+        //
+        // The user and host components are not used.
+        // The password is used with the PRAGMA KEY statement.
```

Comment:
the whole thing above should be a function comment so it appears in the docs (above `pub fn` and using `///` instead of `//`)

Reply:
Good to know.  I'll make the change.